src/includes/sourcemaps-create-react-app.mdx

```diff
@@ -43,15 +43,11 @@ For more info on `sentry-cli` configuration visit the [Sentry CLI configuration
 
 Make sure `sentry-cli` is configured for your project. For that you can use environment variables:
 
-<SignInNote />
-
 <OrgAuthTokenNote />
 
 ```bash {filename:.env}
 SENTRY_ORG=___ORG_SLUG___
 SENTRY_PROJECT=___PROJECT_SLUG___
-# Auth tokens can be obtained from
-# https://sentry.io/settings/auth-tokens/
```

Comment:
This is not in the code example anymore (the link also gets quite long...) instead the link is only in the note above.